Core/src/org/sleuthkit/autopsy/report/ReportHTML.java

```diff
@@ -1128,24 +1147,23 @@ private void writeSummary() {
     }
 
     private String prepareThumbnail(AbstractFile file) {
-        File thumbFile = ImageUtils.getCachedThumbnailFile(file, ImageUtils.ICON_SIZE_MEDIUM);
-        if (thumbFile.exists() == false) {
+        BufferedImage bufferedThumb = ImageUtils.getThumbnail(file, ImageUtils.ICON_SIZE_MEDIUM);
+        File thumbFile = Paths.get(thumbsPath, file.getName() + ".png").toFile();
+        if (bufferedThumb == null) {
             return null;
         }
-        File to = new File(thumbsPath);
-        FileObject from = FileUtil.toFileObject(thumbFile);
-        FileObject dest = FileUtil.toFileObject(to);
         try {
-            FileUtil.copyFile(from, dest, thumbFile.getName(), "");
+            ImageIO.write(bufferedThumb, "png", thumbFile);
         } catch (IOException ex) {
-            logger.log(Level.SEVERE, "Failed to write thumb file to report directory.", ex); //NON-NLS
-        } catch (NullPointerException ex) {
-            logger.log(Level.SEVERE, "NPE generated from FileUtil.copyFile, probably because FileUtil.toFileObject returned null. \n" +
-                    "The File argument for toFileObject was " + thumbFile + " with toString: " + thumbFile.toString() + "\n" +
-                    "The FileObject returned by toFileObject, passed into FileUtil.copyFile, was " + from, ex);
+            logger.log(Level.WARNING, "Failed to write thumb file to report directory.", ex); //NON-NLS
+            return null;
         }
-
-        return THUMBS_REL_PATH + thumbFile.getName();
+        if (thumbFile.exists()
```

Comment:
yes that was the other part I didn't understand why auto format was insisting on it being that way.